integration/update_test.go

```diff
@@ -228,15 +228,6 @@ func TestUpdateReplaceDocuments(t *testing.T) {
 			expectedFilter: bson.D{{"_id", "document-composite"}},
 			expected:       bson.D{{"_id", "document-composite"}, {"replacement-value", int32(1)}},
 		},
-		// TODO: https://github.com/FerretDB/FerretDB/issues/1000
```

Comment:
This test is basically the same as "ReplaceEmptyDocument" in update_compat_test.go